src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -6,99 +6,56 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 
 import static org.hamcrest.Matchers.arrayContainingInAnyOrder;
 import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
 
 public class DatabaseManagerTest {
-    private static final String TEST_DB_NAME = "test";
-    private static final String DB_USER_NAME = "sqlcmd";
+    private static final String ADMIN_DB_NAME = "postgres";
+    private static final String TEST_DB_NAME = "sqlcmd_test";
+    private static final String DB_ADMIN_LOGIN = "postgres";
+    private static final String DB_ADMIN_PASSWORD = "postgres";
+    private static final String DB_USER_LOGIN = "sqlcmd";
     private static final String DB_USER_PASSWORD = "sqlcmd";
     private static final String JDBC_URL = "jdbc:postgresql://localhost:5432/";
-    private static final String TEST_TABLE_NAME = "test_table";
     private static Connection connection;
 
-    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
-    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
-    private PrintStream originalOut;
-    private PrintStream originalErr;
     private DatabaseManager databaseManager;
 
     @BeforeClass
     public static void setTestingEnvironment() throws SQLException {
-        connection = DriverManager.getConnection(
-                JDBC_URL, DB_USER_NAME, DB_USER_PASSWORD);
+        connection = DriverManager.getConnection(JDBC_URL + ADMIN_DB_NAME, DB_ADMIN_LOGIN, DB_ADMIN_PASSWORD);
         executeSqlQuery("DROP DATABASE IF EXISTS " + TEST_DB_NAME);
-        executeSqlQuery("CREATE DATABASE " + TEST_DB_NAME);
+        executeSqlQuery("CREATE DATABASE " + TEST_DB_NAME + " OWNER =" + DB_USER_LOGIN);
         connection.close();
-        connection = DriverManager.getConnection(
-                JDBC_URL + TEST_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD);
+        connection = DriverManager.getConnection(JDBC_URL + TEST_DB_NAME, DB_ADMIN_LOGIN, DB_ADMIN_PASSWORD);
+        recreateDbSchema();
+        connection.close();
+        connection = DriverManager.getConnection(JDBC_URL + TEST_DB_NAME, DB_USER_LOGIN, DB_USER_PASSWORD);
     }
 
     @Before
-    public void setUpStreamsAndDbManager() throws SQLException {
+    public void setUpDbManager() throws SQLException {
         recreateDbSchema();
         databaseManager = new DatabaseManager();
-        databaseManager.connect(TEST_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD);
-        originalOut = System.out;
-        originalErr = System.err;
-        System.setOut(new PrintStream(outContent));
-        System.setErr(new PrintStream(errContent));
+        databaseManager.connect(TEST_DB_NAME, DB_USER_LOGIN, DB_USER_PASSWORD);
     }
 
     @After
-    public void cleanUpStreams() throws SQLException {
+    public void closeDbManagerConnection() throws SQLException {
```

Comment:
closeConnection(), DbManager is redundant (see previous remark)

Reply:
There are already closeConnection, and they should be separated

Reply:
Well, it could be simple close()  or closeResources()... Whatever it's not principal, main thing is that method has understandable name.